.devcontainer/README.md

```
@@ -0,0 +1,88 @@
+- **URL**: Port 9001
+- **Username**: `minioadmin`
+- **Password**: `minioadmin`
+
+### PostgreSQL
+- **Database**: `appdb`
+- **Username**: `appuser`
+- **Password**: `apppass`
+
+## 📁 File Structure
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)